release/rebuild_bom_files.sh

```diff
@@ -0,0 +1,100 @@
+#!/bin/bash
+
+source _bom.sh
+source _git.sh
+source _liferay_common.sh
+source _package.sh
+source _product.sh
+source _publishing.sh
+
+function check_usage {
+	if [ -z "${LIFERAY_RELEASE_GIT_REF}" ]
+	then
+		print_help
+	fi
+
+	if [ -z "${LIFERAY_RELEASE_PRODUCT_NAME}" ]
+	then
+		LIFERAY_RELEASE_PRODUCT_NAME=dxp
+	fi
+
+	_BUILD_TIMESTAMP=$(date +%s)
+
+	_RELEASE_TOOL_DIR=$(dirname "$(readlink /proc/$$/fd/255 2>/dev/null)")
+
+	lc_cd "${_RELEASE_TOOL_DIR}"
+
+	mkdir -p release-data
+
+	lc_cd release-data
+
+	_RELEASE_ROOT_DIR="${PWD}"
+
+	_BUILDER_SHA=$(git rev-parse HEAD)
```

Comment:
@pgydev, will this command work correctly since you're in the directory `release-data`?

Please see below:

![image](https://github.com/natocesarrego/liferay-docker/assets/6729287/ab945768-fda9-44a4-aaeb-ab3fc48d13c5)

Besides, this variable `_BUILDER_SHA` isn't used in the script :disappointed: